test/fixtures/models/player.js

```diff
@@ -14,6 +14,10 @@ export default (sequelize, DataTypes) => {
         models.Player.belongsTo(models.Team, {
           foreignKey: { name: 'teamId' },
         });
+        models.Player.belongsTo(models.Master, {
+          foreignKey: 'coachId',
+          as: 'Coach',
```

Comment:
Can you not use Teams and Players? Just so we don't get an overly-complex mock schema?